gems/smithy/lib/smithy/vise/endpoints.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative 'endpoints/built_in_binding'
+require_relative 'endpoints/function_binding'
+
+module Smithy
+  module Vise
+    # Endpoint Rules related metadata
+    # TODO: Where does this actually belong.
+    # it could be in vise because its related to understanding the model
```

Comment:
I think vise makes sense if you're just using it to parse endpoint rules into objects/hash/lists to make it easier for views to render.